src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseSource.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data.Common;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Data
+{
+    public sealed class DatabaseSource
```

Comment:
Can we slap a few quick xml summary comments to the public APIs?

Also, I assume we can unseal this later, if necessary. Should be fine to seal it for now.

Reply:
Right. Unsealing is safe, sealing is not

Reply:
Added comments.